### PR TITLE
[CS-1194] Fix sending of SPOA

### PR DESCRIPTION
**Pull request by DominicSherman**
Need to handle instances of hard coding to check for `ETH` rather than the given native token for the network

### CODE REVIEW

File: src/handlers/web3.js [hunk 1]
```diff
@@ -311,8 +311,9 @@ export const getTransferTokenTransaction = async transaction => {
  * @param {Object} transaction { asset, from, to, amount, gasPrice }
```
Comment (topwebtek7) [inline src/handlers/web3.js:311]: @DominicSherman could you update this as well cause params are changed?

File: src/handlers/web3.js [hunk 2]
```diff
@@ -381,6 +382,7 @@ export const getDataForNftTransfer = (from, to, asset) => {
  */
```
Comment (topwebtek7) [inline src/handlers/web3.js:384]: same for this as well

**Review by topwebtek7 — APPROVED** [review]
left a couple small comments, others looks good to me

File: src/handlers/web3.js (hunk 1 above)
Comment (DominicSherman) [reply src/handlers/web3.js:311]: Updated

Comment (DominicSherman) [reply src/handlers/web3.js:311]: Updated

**Pull request merged**